.gitignore

```diff
@@ -13,8 +13,10 @@ lib/json
 lib/protobuf
 
 .idea/
+.vscode/
 
 # Generated files
+swift/TrustWalletCore.xcodeproj
```

Comment:
This is already in here. Let's not add a duplicate entry.

Reply:
> This is already in here. Let's not add a duplicate entry.

done